Doc/library/tkinter.rst

```diff
@@ -799,6 +799,10 @@ reference to the image. When the last Python reference to the image object is
 deleted, the image data is deleted as well, and Tk will display an empty box
 wherever the image was used.
 
+.. seealso::
+
+    The `Pillow package <http://python-pillow.org/>`_ adds support for
```

Comment:
The note above (Doc/distutils/introduction.rst) tells about misleading of calling it a package.

Reply:
If I understand correctly, the note above is detailing Distutils-specific terminology. If you go up a little, the definition of "package" in the General Python terminology section accurately describes Pillow.

Reply:
Okay then.